point_viewer_proto_rust/src/proto.proto

```diff
@@ -75,23 +75,51 @@ message NodeId {
   int64 deprecated_index = 2;
 }
 
-message Node {
-  enum PositionEncoding {
+enum PositionEncoding {
     INVALID = 0;
     Uint8 = 1;
     Uint16 = 2;
     Float32 = 3;
     Float64 = 4;
-  }
+}
 
+
+message Node {
   PositionEncoding position_encoding = 2;
   int64 num_points = 3;
   NodeId id = 4;
 }
 
+enum AttributeType{
+    I64 = 0;
+    U64 = 1;
+    F32 = 2;
+    F64 = 3;
+    U8Vec3 = 4;
+    F64Vec3 = 5;
+}
+
+message AttributeDescription{
+  string name = 1;
+  AttributeType data_type = 2;
+}
+
+message S2Cell {
+  PositionEncoding position_encoding = 1;
```

Comment:
Shall we add this later, once we use it, or just ignore it for now?

Reply:
as every field is optional, i think this mirrors also node contents as well and will hopefully be soon in use